src/main/kotlin/com/terraformation/backend/search/table/SpeciesTable.kt

```diff
@@ -42,7 +44,11 @@ class SpeciesTable(tables: SearchTables) : SearchTable() {
           timestampField("checkedTime", SPECIES.CHECKED_TIME),
           textField("commonName", SPECIES.COMMON_NAME),
           enumField("conservationCategory", SPECIES.CONSERVATION_CATEGORY_ID, localize = false),
-          booleanField("endangered", SPECIES.ENDANGERED),
+          booleanField(
+              "endangered",
+              DSL.case_(SPECIES.CONSERVATION_CATEGORY_ID)
+                  .`when`(ConservationCategory.Endangered, true)
```

Comment:
Does this include the whole series of categories of Vulnerable or worse, or only Endangered?

Reply:
Got it, thanks for linking the discussion